.ci/.docker-images.yml

```diff
@@ -0,0 +1,336 @@
+---
+# Dockerfile build config for .github/workflows/publish-docker-images.yml
+# Schema: .ci/.docker-images.schema.json
+# See docs/INTERNAL_DOCKER_IMAGES.md for further information.
+
+templates:
+
+  - &apm-agent-library-docker-images
+    repository: "elastic/apm-pipeline-library"
+    working_directory: ".ci/docker"
+
+  - &opbeans-docker-images
+    tag: "daily"
+
+  - &apm-agent-python-docker-images
+    name: "apm-agent-python"
+    repository: "elastic/apm-agent-python"
+    working_directory: "tests"
+
+  - &apm-agent-nodejs-docker-images
+    name: "apm-agent-nodejs"
+    repository: "elastic/apm-agent-nodejs"
+    working_directory: ".ci/docker/node-container"
+
+  - &apm-agent-ruby-docker-images
+    name: "apm-agent-ruby"
+    repository: "elastic/apm-agent-ruby"
+    working_directory: "spec"
+
+  - &apm-agent-rum-docker-images
+    repository: "elastic/apm-agent-rum-js"
+
+  - &observability-dev-docker-images
+    repository: "elastic/observability-dev"
+
+images:
+
+  # observability-test-environments Docker Images
+
+  - name: "functional-opbeans"
+    repository: "elastic/observability-test-environments"
+    working_directory: "tests"
+    build_script: "docker build --force-rm -t ${REGISTRY}/${PREFIX}/${NAME}:${TAG} ${NAME}"
+    push_script: "docker push ${REGISTRY}/${PREFIX}/${NAME}:${TAG}"
+
+## This is also failing in Jenkins
+##  https://apm-ci.elastic.co/job/apm-shared/job/docker-images/job/oracle-instant-client/
+#  - name: "azure-vm-tools"
+#    repository: "elastic/azure-vm-extension"
+#    working_directory: ".ci/docker/azure-vm-tools"
+
+  # observability-robots Docker Images
+
+  - name: "picklesdoc"
+    repository: "elastic/observability-robots"
+    build_script: "make build"
+    push_script: "make push"
+    working_directory: "apps/pickles"
+
+## This is also failing on Jenkins
+## https://apm-ci.elastic.co/job/apm-shared/job/docker-images/job/test-plans/
+#  - name: "test-plans"
+#    repository: "elastic/observability-robots"
+#    build_script: "make build"
+#    push_script: "make push"
+#    working_directory: "apps/test-plans"
+
+  - name: "obs-jenkins-heartbeat"
+    repository: "elastic/observability-robots"
+    prepare_script: |
+      pip3 install pyyaml
+      python3 ./generate_heartbeat_configs.py
+    working_directory: "apps/beats/heartbeat"
+
+  # observability-dev Docker Images
+
+  - <<: *observability-dev-docker-images
+    name: "bandstand"
+    working_directory: "apps/automation/bandstand"
+
+  - <<: *observability-dev-docker-images
+    name: "slack-bridge-hey-apm"
+    working_directory: "tools/report-bridge"
+
+  - <<: *observability-dev-docker-images
+    name: "flakey"
+    working_directory: "apps/automation/jenkins-toolbox"
```

Comment:
This project is deprecated. We can actually remove this.

Reply:
[0b14593](https://github.com/elastic/apm-pipeline-library/pull/1986/commits/0b145931c85b3d7aad481f76d398081f1fbb3cfa)